upgrade/1.2/scripts/common/ncn-rebuild-common.sh

```diff
@@ -308,6 +298,16 @@ if [[ $target_ncn != ncn-s* ]]; then
     } >> ${LOG_FILE} 2>&1
 fi 
 
+# force time to sync immediately
+{
+  ssh "${target_ncn}" -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null chronyc makestep
+  sleep 1
+  in_sync=$(ssh "${target_ncn}" timedatectl | awk /synchronized:/'{print $NF}')
+  if [[ "$in_sync" = "no" ]]; then
```

Comment:
double equal?

Reply:
I read up on that.  `==` is a bash-ism for `=`.  I'm being inconsistent here though, so I'll fix.